### PR TITLE
Fix documentation format for tuples in VB too

### PR DESCRIPTION
**Pull request by jcouv**
Corresponding change to https://github.com/dotnet/roslyn/pull/18380 for VB

@dotnet/roslyn-compiler for review.

### CODE REVIEW

**Comment by jcouv** [comment]
@dotnet-bot test windows_eta_open_prtest please

**Comment by jcouv** [comment]
@dotnet-bot test windows_release_vs-integration_prtest please

File: src/Compilers/VisualBasic/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.vb [hunk 1]
```diff
@@ -127,6 +127,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Function
 
             Public Overrides Function VisitNamedType(symbol As NamedTypeSymbol, builder As StringBuilder) As Object
+                If symbol.IsTupleType Then
+                    Return VisitNamedType(DirectCast(symbol, TupleTypeSymbol).UnderlyingNamedType, builder)
+                End If
+
                 If symbol.ContainingSymbol IsNot Nothing AndAlso symbol.ContainingSymbol.Name.Length <> 0 Then
```
Comment (AdamSpeight2008) [inline src/Compilers/VisualBasic/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.vb]: Can be rewritten as `If symbol.ContainingSymbol?.Name.Length <> 0 Then`

**Comment by jcouv** [comment]
@dotnet-bot test windows_release_vs-integration_prtest please

**Comment by jcouv** [comment]
@AlekseyTs for a second code review. Thanks

**Comment by jcouv** [comment]
@AdamSpeight2008 I'm not sure that works with mode Strict.

**Pull request merged**